geode-lucene/src/test/java/org/apache/geode/cache/lucene/internal/PartitionedRepositoryManagerJUnitTest.java

```diff
@@ -139,12 +134,16 @@ protected void createIndexAndRepoManager() {
     when(indexForPR.getAnalyzer()).thenReturn(new StandardAnalyzer());
     when(indexForPR.getCache()).thenReturn(cache);
     when(indexForPR.getRegionPath()).thenReturn("/testRegion");
-    prRoot = Mockito.mock(LocalRegion.class);
+
+    prRoot = Mockito.mock(DistributedRegion.class);
```

Comment:
I was able to run all tests and all validations pass, but it isn't obvious to me why we no longer need to mock PartitionedRegionHelper and instead are mocking cache.createVMRegion().  Do you mind explaining just for my own education?